iotileship/iotile/ship/scripts/iotile_ship.py

```diff
@@ -0,0 +1,93 @@
+"""Command line script to load and run a sensor graph."""
```

Comment:
For python 3 compatibility all files should start with:

```
from __future__ import (unicode_literals, absolute_import, print_function)
from builtins import str
```

This will make sure all of the easy portability mistakes get caught and you don't need to prefix all of your strings with `u`